3p/iframe-messaging-client.js

```diff
@@ -100,6 +100,19 @@ export class IframeMessagingClient {
    */
   setupEventListener_() {
     listen(this.win_, 'message', event => {
+      try {
+        if (this.hostWindow_ && !this.hostWindow_.postMessage &&
```

Comment:
hostWindow_ does not allow nullable and existence of contentWindow should be sufficient for checking the iframe case so I believe this can be collapsed to:

this.hostWindow_ = this.hostWindow_.contentWindow || this.hostWindow_;

In fact, I think you should be able to do this all within the constructor allowing for this.hostWindow_ to be a const.  Any reason you expect it to throw here?